scraping/script.ipynb

```diff
@@ -90,35 +230,37 @@
     "\n",
     "\n",
     "def main():\n",
-    "    base_url = \"https://www.foodnetwork.com/recipes/food-network-kitchen/123\"  # Starting with '123'\n",
+    "    counter = 0\n",
+    "    base_url = \"https://en.wikibooks.org/wiki/Category:Recipes\"\n",
     "    urls = generate_urls(base_url)\n",
+    "    print(f\"Found {len(urls)} urls.\")\n",
+    "    print(f\"{urls}\")\n",
     "    recipe_index = {}\n",
     "\n",
     "    for url in urls:\n",
     "        print(f\"Processing {url}...\")\n",
-    "        response = requests.get(url)\n",
-    "        soup = BeautifulSoup(response.text, 'html.parser')\n",
     "\n",
     "        # Extract recipes from current page and scrape them\n",
     "        recipes = get_recipes_from_page(url)\n",
     "        print(f\"Found {len(recipes)} recipes.\")\n",
+    "        print(f\"{recipes}\")\n",
     "\n",
     "        # Iterate over each URL to access and extract information\n",
     "        for recipe in recipes:\n",
-    "            recipe_id = str(uuid.uuid4())\n",
     "            details = scrape_recipe_details(recipe)\n",
-    "            recipe_index[recipe_id] = details['title']\n",
-    "            write_json(details, f\"{recipe_id}.json\")\n",
+    "            recipe_index[counter] = details\n",
+    "            counter += 1\n",
+    "            print(f\"{recipe} has been processed.\")\n",
     "        print(f\"{url} has been processed.\")\n",
     "    \n",
     "    print(\"All URLs have been processed.\")\n",
     "    \n",
     "    # Writing the index file and ingredient/category accumulations\n",
-    "    write_json(recipe_index, \"recipe_index.json\")\n",
+    "    write_json(recipe_index, \"recipes.json\")\n",
     "    write_json(list(all_ingredients), \"all_ingredients.json\")\n",
     "    write_json(list(all_categories), \"all_categories.json\")\n",
     "\n",
-    "# main() TODO: UNCOMMENT TO RUN"
+    "main() # TODO: UNCOMMENT TO RUN"
```

Comment:
Running this script will print the results on the output console right?

Reply:
Yes it will ! Since I am using a jupyter notebook to run the code it will print out the results in an output cell directly under the cell of my code.

Reply:
Ok great!